Mathlib/Tactic/Rename.lean

```diff
@@ -11,7 +11,9 @@ open Lean Elab.Tactic Meta
 
 syntax renameArg := term " => " ident
 
-/-- `rename' h => hnew` renames the hypothesis named `h` to `hnew`. -/
+/-- `rename' h => hnew` renames the hypothesis named `h` to `hnew`.
+To rename several hypothesis, use `rename' h₁ => h₁new, h₂ => h₂new`.
+You can use `rename' a => b, b => a` to swap two variables. -/
```

Comment:
Wait, this works?!?  That was certainly not the intention (or the way the code is written).

Reply:
This does work; I added a test case.

Reply:
Okay, then my hint was too subtle.  It doesn't work in half of the cases:
```lean
example (b : Nat) (a : Int) : Nat × Int := by
  rename' a => b, b => a
  exact (a, b)
```

Reply:
Oops. :-)